streamer/src/nonblocking/quic.rs

```diff
@@ -720,8 +706,38 @@ async fn setup_connection(
     if let Ok(connecting_result) = res {
         match connecting_result {
             Ok(new_connection) => {
+                debug!("Got a connection {from:?}");
```

Comment:
This should not be moved as in the caller there was other error conditions it is easier to debug an correlate.